src/dotnet-publish-iis/project.json

```diff
@@ -14,12 +14,21 @@
     "Microsoft.Extensions.Configuration.Json": "1.0.0-*",
     "Microsoft.DotNet.ProjectModel": "1.0.0-*",
     "Microsoft.DotNet.Cli.Utils": "1.0.0-*",
-    "NETStandard.Library": "1.5.0-*",
     "System.Diagnostics.Process": "4.1.0-*"
   },
 
   "frameworks": {
-    "dnxcore50": {
+    "netstandard1.5": {
```

Comment:
If this is an app perhaps it should be `netcoreapp1.0`. That should also eliminate the need for the imports.